src/context/loadIsoVolumes.tsx

```diff
@@ -17,24 +16,3 @@ export const loadIsoVolumes = async (
 
   return remoteImages;
 };
-
-export const loadVolumes = async (
```

Comment:
I think we can't remove this function since we need to support `lxd 5.0/stable`.

Reply:
No adjustments needed for tests since the api call should always be using the right endpoints